sdk/appconfiguration/azure-appconfiguration-provider/samples/aad_sample.py

```diff
@@ -7,26 +7,34 @@
 from azure.appconfiguration.provider import load, SettingSelector
 import os
 from sample_utilities import get_authority, get_audience, get_credential
+from azure.core.exceptions import ClientAuthenticationError
 
-endpoint = os.environ.get("AZURE_APPCONFIG_ENDPOINT")
+endpoint = os.environ.get("APPCONFIGURATION_ENDPOINT_STRING")
 authority = get_authority(endpoint)
 audience = get_audience(authority)
 credential = get_credential(authority)
 
 # Connecting to Azure App Configuration using AAD
-config = load(endpoint=endpoint, credential=credential)
-
-print(config["message"])
+try:
```

Comment:
Why need this?

Reply:
There's `ClientAuthenticationError` with endpoint. I don't know if provider samples are working as there doesn't have a chance to run in our pipeline. I'll leave them to fix it.